Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "glx"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
Since this library just exports the results of gl_generator, we should do a minor version bump when gl_generator does a minor version bump.